kip-0039.md

```diff
@@ -0,0 +1,161 @@
+---
+KIP: "0039"
+Title: kadena_getNetwork_v1 API
+Author: Danillo Felixdaal, Bart Huijgen and Albert Groothedde
+Status: Draft
+Type: Standard
+Category: Interface
+Created: 2025-02-17
+---
+
+# kadena_getNetwork_v1 API
+
+## Abstract
+
+This KIP standardizes the `kadena_getNetwork_v1` method for retrieving
+information about the **currently active network** from a Kadena wallet. The
+method provides consistent network details to dApps, ensuring they can correctly
+adjust to the appropriate blockchain environment.
+
+## Motivation
+
+A uniform approach to obtaining network information simplifies dApp development
+by ensuring that network details are provided in a consistent format across
+different wallet implementations. This standardization will:
+
+- Provide reliable network details (such as network name and networkId) to
+  dApps.
+- Simplify integration and reduce potential mismatches between wallet and dApp
+  network settings.
+- Enhance the overall user experience by ensuring clear network context.
+
+The dapp can use the URL to make /local calls from the network.
```

Comment:
This is the first mention of `URL` in the document. This needs an transition sentence between the previous points (benefits of this method) to the current point (dApps can use the URL).

Maybe something like:
```suggestion
The wallet will also return a list of endpoint URLs for the active network. The dApp can use these URLs to make `/local` calls to the network.
```

and/or move this entire URL section after the Response type definition in the `Specification` section.

(similar comment as in KIP 40)